src/App.tsx

```diff
@@ -17,18 +59,32 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                setOptionsValue={setOptionsValue}
+                setInputValue={setInputValue}
+                inputValue={inputValue}
+              />
             </div>
+            {loading && <Loader />}
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {!loading && todos.length > 0 && (
+                <TodoList
+                  filteredTodos={filteredTodos}
```

Comment:
The prop `filteredTodos` is misspelled as `fillteredTodos`. Please correct it to `filteredTodos` to match the variable name used in the `useMemo` hook.